### DISCUSSION TITLE
What happened to certificate manager?

### DISCUSSION THREAD

**Discussion by crass** [Ylianst/MeshCentral#4777] [Q&A] [answered]
Watching this tutorial on [Meshcentral and TLS](https://youtu.be/2WYanQ11gYE?t=377) it looks like version 0.4.0 had a certificate manager. On a version 0.9.8 install, I see no such certificate manager gui as was shown in the video. Do I need to explicitly enable it, or was that functionality removed?

**Comment by jsastriawan** [comment]
This is available on Meshcommander Node Webkit version. It may not be available on webserver edition.

**Reply by crass** [reply]
Where is the webkit version, I don't see it [here](https://meshcentral.com/info/downloads.html). And does this mean that it would be available if I was using the webserver edition with a webkit based browser?

**Comment by si458** [comment, marked as answer]
meshcommander and meshcentral are 2 different products
meshcommander is for AMT management
meshcentral is for remote support
https://www.meshcommander.com/meshcommander - link to the meshcommander docs and install etc

**Reply by crass** [reply]
Ahh right, got them mixed up. But again, were is the webkit version? Its not clearly (to a new comer) identified at https://www.meshcommander.com/meshcommander. Could it be the windows install file?

**Reply by si458** [reply]
> Ahh right, got them mixed up. But again, were is the webkit version? Its not clearly (to a new comer) identified at https://www.meshcommander.com/meshcommander. Could it be the windows install file?

Yes it is, webkit is just a framework that's used to make the exe run the html files

**Reply by crass** [reply]
Ok, so its windows only. Thanks for clearing that up. Might be less confusing to note on the website that its a standalone webkit-based version of MeshCommander.

**Reply by crass** [reply]
For posterity, [this documents](https://ylianst.github.io/MeshCentral/meshcmd/#meshcommander) that MashCommander in webserver mode does not have the certificate management.
